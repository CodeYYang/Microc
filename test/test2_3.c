void main() {
  int n=5;
  int x;
  x= n>2?x=1:x=2;
  print("%d",x);
}