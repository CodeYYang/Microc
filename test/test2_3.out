24 19 0 5 25 13 0 5 12 15 1 13 0 1 1 13 11 13 0 1 1 0 1 12 16 33 13 0 1 1 0 2 12 10 7 12 15 -1 13 0 1 1 11 22 21 2